### PR TITLE
This allows inherited TSC to be populated with proper Parameters

### PR DESCRIPTION
**Pull request by sconde**
In Drekar's TSC, we needed the stepper info. Making this small change in
Tempus prevents a lot of "hacks" in Drekar. I also suspect others might
find it useful

@trilinos/tempus

## Related Issues

* Closes 
* Blocks [Drekar Tempus integration](https://github.com/rppawlo/DrekarBase/pull/1)
* Is blocked by 
* Follows 
* Precedes 
* Related to 
* Part of 
* Composed of 

## How Has This Been Tested?

[Tests log](https://github.com/trilinos/Trilinos/files/1536205/all_test.txt)
Note see [Drekar Gitlab Issue#87](https://gitlab.sandia.gov/EM-Plasma/Drekar/issues/87) for thesingle test failure.

## Checklist

- [ ] My commit messages mention the appropriate GitHub issue numbers.
- [x] My code follows the code style of the affected package(s).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [code contribution guidelines](../blob/master/CONTRIBUTING.md) for this project.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] No new compiler warnings were introduced.
- [ ] These changes break backwards compatibility.

### CODE REVIEW

**Comment by jmgate** [comment]
@sconde, do be sure you fill out the relevant portions of the pull request template when you create one.  To do so now, you can edit the description above.

**Comment by sconde** [comment]
@ccober6 @rppawlo could you please review this PR

File: packages/tempus/src/Tempus_IntegratorBasic_impl.hpp [hunk 1]
```diff
@@ -257,6 +248,16 @@ void IntegratorBasic<Scalar>::setTimeStepControl(
     timeStepControl_ = Teuchos::null;
```
Comment (rppawlo) [inline packages/tempus/src/Tempus_IntegratorBasic_impl.hpp]: I think this line is superfluous and should be removed. It's not part of your change set, but maybe you could remove as part of this push?

**Comment by ccober6** [comment]
Agree with Roger; that line is "superfluous and should be removed".  Otherwise looks good!

**Pull request closed without merging**